src/desktop/apps/auction/components/layout/__tests__/ConfirmRegistrationModal.jest.tsx

```diff
@@ -76,7 +76,7 @@ describe("Confirm Registration Modal", () => {
       it("shows a qualified message", () => {
         const { wrapper } = renderTestComponent({
           Component: ConfirmRegistrationModal,
-          options: { renderMode: "render" },
+          options: { renderMode: "mount" },
```

Comment:
`render` did not work and rendered nothing with the `PostRegistrationModal` component in Reaction. Not sure why it's working with `mount` even after reading https://github.com/airbnb/enzyme/issues/465#issuecomment-227697726.